### ISSUE TITLE
[kube-up][scale-out]: code dead lock in function start-mizar-scaleout

### ISSUE THREAD

**Issue by sonyafenge** [CentaurusInfra/arktos#1361]
**What happened**:
there are dead lock in function start-mizar-scaleout
```
function start-mizar-scaleout {
...
echo "Waiting until nodes RPs are ready"
until [ -f "/etc/srv/kubernetes/kube-scheduler/rp-kubeconfig-1" ]; do
   sleep 5
done
...
}
```
start-cluster-networking -> start-mizar -> start-mizar-scaleout running only on TP server. 
Scale-out starting process:
TP Servers -> RP servers -> proxy

when start mizar, RP servers are not  ready. this until will be dead lock. 

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
Added comments and found the logs in  /var/log/master-init.log
```
echo "Waiting until nodes RPs are ready"
  until [ -f "/etc/srv/kubernetes/kube-scheduler/rp-kubeconfig-1" ]; do
    echo "until dead lock"
    date
    sleep 5
  done
  date
  kubectl create configmap system-source --namespace=kube-system --from-literal=name=arktos --from-literal=company=futurewei

```

```
Deploying mizar daemonset
daemonset.apps/mizar-daemon created
Waiting until nodes RPs are ready
until dead lock
Tue Feb 15 03:20:49 UTC 2022
until dead lock
Tue Feb 15 03:20:54 UTC 2022
until dead lockTue Feb 15 03:20:59 UTC 2022
until dead lock
Tue Feb 15 03:21:04 UTC 2022
Deploying mizar daemonset
daemonset.apps/mizar-daemon created
Waiting until nodes RPs are ready
until dead lock
Tue Feb 15 03:20:49 UTC 2022
...
until dead lock
Tue Feb 15 03:27:45 UTC 2022
until dead lock
Tue Feb 15 03:27:50 UTC 2022
Tue Feb 15 03:27:55 UTC 2022
configmap/system-source created
```

Repro steps:
```
$ unset KUBE_GCE_MASTER_PROJECT KUBE_GCE_NODE_PROJECT KUBE_GCI_VERSION  KUBE_GCE_MASTER_IMAGE  KUBE_GCE_NODE_IMAGE KUBE_CONTAINER_RUNTIME NETWORK_PROVIDER
$ export KUBEMARK_NUM_NODES=100 NUM_NODES=1 SCALEOUT_CLUSTER=true SCALEOUT_TP_COUNT=1 SCALEOUT_RP_COUNT=1 RUN_PREFIX=sonyaperf2-021422
$ export MASTER_DISK_SIZE=500GB MASTER_ROOT_DISK_SIZE=500GB KUBE_GCE_ZONE=us-west2-b MASTER_SIZE=n1-highmem-32 NODE_SIZE=n1-highmem-16 NODE_DISK_SIZE=500GB GOPATH=$HOME/go KUBE_GCE_ENABLE_IP_ALIASES=true KUBE_GCE_PRIVATE_CLUSTER=true CREATE_CUSTOM_NETWORK=true KUBE_GCE_INSTANCE_PREFIX=${RUN_PREFIX} KUBE_GCE_NETWORK=${RUN_PREFIX} ENABLE_KCM_LEADER_ELECT=false ENABLE_SCHEDULER_LEADER_ELECT=false ETCD_QUOTA_BACKEND_BYTES=8589934592 SHARE_PARTITIONSERVER=false LOGROTATE_FILES_MAX_COUNT=200 LOGROTATE_MAX_SIZE=200M KUBE_ENABLE_APISERVER_INSECURE_PORT=true KUBE_ENABLE_PROMETHEUS_DEBUG=true KUBE_ENABLE_PPROF_DEBUG=true TEST_CLUSTER_LOG_LEVEL=--v=2 HOLLOW_KUBELET_TEST_LOG_LEVEL=--v=2 GCE_REGION=us-west2-b NETWORK_PROVIDER=mizar
$ ./cluster/kube-up.sh
```
**Anything else we need to know?**:

if ubuntu 18.04, script will stop on this loop.

**Environment**:
- Arktos version (use `kubectl version`):
https://github.com/CentaurusInfra/arktos/commit/2b6855fa7b204870620421cb1c533b182acbc812

- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by vinaykul** [comment]
Per email and meeting discussion, this is behaving as intended. Sonya will try to relocate when Mizar is started for scaleoup

**Comment by vinaykul** [comment]
Please try increasing the timeout from 300 to 600 or more for 18.04 to accommodate the time taken in updating the kernel for Mizar to work.

**Comment by sonyafenge** [comment]
fixed with PR #1363
